dist/node/canvg.js

```diff
@@ -404,8 +404,8 @@
 	  svg.Property.prototype.toMilliseconds = function () {
 	    if (!this.hasValue()) return 0;
 	    var s = this.value + '';
-	    if (s.match(/s$/)) return this.numValue() * 1000;
 	    if (s.match(/ms$/)) return this.numValue();
+	    if (s.match(/s$/)) return this.numValue() * 1000;
 	    return this.numValue();
```

Comment:
i think i had this wrong to begin with. https://svgwg.org/specs/animations/#ClockValueSyntax seems like the default is `s` so we should multiply this line by `1000` as well. if you can, add the link above the method so future devs can implement the rest of the spec if they have time.

Reply:
not in the `dist` file obviously but the one below :)